File: Python/repetitivas/tabuada.py
# Exercício: Tabuada
# Objetivo: exibir a tabuada de um número de 1 a 10.

n = int(input("Deseja a tabuada para qual valor? "))

for i in range(1, 11):
    tabuada = n * i
    print(f"{n} X {i} = {tabuada}")

File: Java/Estruturas Repetitivas/tabuada.java
// Exercício: Tabuada
// Objetivo: calcular e exibir a tabuada de um número informado pelo usuário.

import java.util.Scanner;

public class tabuada {
    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);

        int n, i, produto;

        System.out.print("Deseja a tabuada para qual valor? ");
        n = sc.nextInt();

        for (i = 1; i <= 10; i++) {
            produto = n * i;
            System.out.println(n + " X " + i + " = " + produto);
        }

        sc.close();
    }
}